doc/man/cockpit.conf.xml

```diff
@@ -196,6 +196,20 @@ ProtocolHeader = X-Forwarded-Proto
     </variablelist>
   </refsect1>
 
+  <refsect1 id="cockpit-conf-session">
+    <title>Session</title>
+    <variablelist>
+      <varlistentry>
+        <term><option>IdleTimeout</option></term>
+        <listitem>
+          <para>Time in minutes after which session expires and user is logged out if no user action
+          has been performed in the given time. Set to <literal>0</literal> to disable session timeout.
+          When not specified, <literal>15</literal> minutes are considered.</para>
```

Comment:
"are considered" sounds a bit strange. "When not specified, the default is 15 minutes."?